src/utils/openOrdex.js

```diff
@@ -132,27 +132,29 @@ Needed:          ${satToBtc(amount)} BTC`);
 }
 
 export async function getAvailableUtxosWithoutInscription({ address, price }) {
-    let dummyUtxo;
-
     const payerUtxos = await getAddressUtxos(address);
     if (!payerUtxos.length) {
         throw new Error(`No utxos found for address ${address}`);
     }
 
+    // We require at least 2 dummy utxos for taker
+    const dummyUtxos = [];
     const potentialDummyUtxos = payerUtxos.filter((utxo) => utxo.value <= DUMMY_UTXO_VALUE);
```

Comment:
To be safe, should also .filter((x) => x.status.confirmed)

Also why do we filter again inside of selectUtxos?

Reply:
Also I think the formula for selectUtxos needs to be updated to account for multiple dummy utxos